docs/content/use-cases/drupal-multisite.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Drupal Multisite"
+weight: 1
+---
+
+>Drupal has a feature which allows separate, independent sites to be served from a single codebase. Each site has its own 
+database, configuration, files and base domain or URL.
```

Comment:
```suggestion
database, configuration, public files folder and base domain or URL.
```

Reply:
Hmmm. As I start tinkering with words, I'm feeling a small rewrite here is in order.